fivetran_provider_async/operators.py

```diff
@@ -106,6 +130,25 @@ def execute(self, context: Context) -> None | str:
                 )
             return None
 
+    def _wait_synchronously(self, last_sync: pendulum.DateTime) -> None:
+        """
+        Wait for the task synchronously.
+
+        It is recommended that you do not use this, and instead set
+        `deferrable=True` and use a Triggerer if you want to wait for the task
+        to complete.
+        """
+        while True:
+            is_completed = self.hook.is_synced_after_target_time(
+                self.connector_id, last_sync, propagate_failures_forward=False, always_wait_when_syncing=True
+            )
+            if is_completed:
+                return None
```

Comment:
nitpick: I think the following would be sufficient in this case
```suggestion
                return
```